libavcodec/cbs_h266.h

```diff
@@ -862,6 +862,14 @@ typedef struct CodedBitstreamH266Context {
     H266RawVPS  *vps[VVC_MAX_VPS_COUNT]; ///< RefStruct references
     H266RawSPS  *sps[VVC_MAX_SPS_COUNT]; ///< RefStruct references
     H266RawPPS  *pps[VVC_MAX_PPS_COUNT]; ///< RefStruct references
+
+    // The currently active parameter sets.  These are updated when any
```

Comment:
vvc spec did not define the active parameter set. 
the parameter set will be active by the H266RawPictureHeader

Reply:
Okay. Will check that.

Reply:
Yeah. Remove it and move the film grain to the h274 sei syntax template also.